ipywidgets/widgets/interaction.py

```diff
@@ -79,10 +80,26 @@ def _widget_abbrev_single_value(o):
 
 def _widget_abbrev(o):
     """Make widgets from abbreviations: single values, lists or tuples."""
-    if isinstance(o, (list, tuple)):
-        if o and all(isinstance(x, string_types) for x in o):
+    if isinstance(o, list):
+        # --------------------------------------------------------------------
+        # Handle deprecated behavior of using lists of length 2 or 3 in place
+        # of tuples to specify slider widget attributes. This will be removed
+        # in ipywidgets 6.0.
+        if len(o) in [2, 3] and all(isinstance(x, Real) for x in o):
```

Comment:
Actually, I guess python optimizes them both into constants.  I'd probably still use `(2,3)` as a matter of style - it's a static list of elements.
```
In [1]: from dis import dis

In [2]: dis(compile("1 in (2,3)", "", "eval"))
  1           0 LOAD_CONST               0 (1)
              3 LOAD_CONST               3 ((2, 3))
              6 COMPARE_OP               6 (in)
              9 RETURN_VALUE

In [3]: dis(compile("1 in [2,3]", "", "eval"))
  1           0 LOAD_CONST               0 (1)
              3 LOAD_CONST               3 ((2, 3))
              6 COMPARE_OP               6 (in)
              9 RETURN_VALUE
```